### ISSUE TITLE
Option to use system font

### ISSUE THREAD

**Issue by Warden20** [vicolo-dev/chrono#16]
It would be awesome to be able to use the system font

**Comment by AhsanSarwar45** [comment]
Hi, currently waiting for [this](https://github.com/flutter/flutter/issues/48381) upstream issue to be resolved before I can add this.

**Comment by dr460nf1r3** [comment]
Might using this plugin be a possibility? https://github.com/flutter/flutter/issues/48381#issuecomment-2028689855

**Comment by AhsanSarwar45** [comment]
> Might using this plugin be a possibility? https://github.com/flutter/flutter/issues/48381#issuecomment-2028689855

Yes, although that seems to work for Android 14 only. But I can add this option to supported devices atleast.
